### ISSUE TITLE
Static file serving on windows, binary vs text

### ISSUE THREAD

**Issue by thejustinwalsh** [toastdriven/itty#9]
I decided to run my itty based web app on windows today and ran into an issue with serving up images using `serve_static_file`.  It appears that things like images must be served as binary or the browser will not consume it properly.

I've worked up a quick hack for my own purposes however my python foo is weak, and I feel someone with more knowledge should attack this issue...
#### Hack

```
def static_file(filename, root=MEDIA_ROOT):
...
...
if content_type(filename).find('text') > -1 or content_type(filename).find('xml') > -1:
  return open(desired_path, 'r').read()

return open(desired_path, 'rb').read()
```

**Comment by toastdriven** [comment]
Sorry for the slow response. Thanks for pointing this out. I'd like to try to work up a cleaner version but will get to this near future.

**Comment by toastdriven** [comment]
Actually, the more I looked at it, the more your patch was fine. Fixed in SHA: d15d5532c2bfada2d127480348b47c1978023847
